django/test/runner.py

```diff
@@ -608,9 +614,13 @@ def add_arguments(cls, parser):
             '-d', '--debug-sql', action='store_true',
             help='Prints logged SQL queries on failure.',
         )
+        try:
+            const_parallel = int(os.environ['DJANGO_TEST_PROCESSES'])
+        except KeyError:
+            const_parallel = 'auto'
         parser.add_argument(
-            '--parallel', nargs='?', default=1, type=int,
-            const=default_test_processes(), metavar='N',
+            '--parallel', nargs='?', const=const_parallel, default=1,
+            type=parallel_type, metavar='N',
```

Comment:
We should update `help` (here and in `runtests.py`).

Reply:
I've updated to "Run tests in parallel. Specify N processes, or "auto" for one process per CPU."